src/main/java/br/com/openenade/api/curso/CursoId.java

```diff
@@ -24,4 +24,20 @@ public CursoId(Long codigoCurso, Modalidade modalidade) {
         this.modalidade = modalidade;
     }
 
+    public Long getCodigoCurso() {
```

Comment:
I think that these methods in CursoId are not necessary.

Reply:
It is for remove certain warnings.

Reply:
Ok.